pkg/cmpstatus/clair.go

```diff
@@ -73,6 +73,30 @@ func (c *Clair) Check(ctx context.Context, reg qv1.QuayRegistry) (qv1.Condition,
 		}, nil
 	}
 
+	// users are able to override the number of replicas. if they do override it to zero
+	// we expect zero replicas to be running.
+	replicas := qv1.GetReplicasOverrideForComponent(&reg, qv1.ComponentClair)
+	scaleddown := replicas != nil && *replicas == 0
+	if scaleddown {
+		if dep.Status.AvailableReplicas == 0 {
+			return qv1.Condition{
+				Type:           qv1.ComponentClairReady,
+				Reason:         qv1.ConditionReasonComponentReady,
+				Status:         metav1.ConditionTrue,
```

Comment:
Base reports itself as Ready: false when it's scaled down, but Clair as Ready: true. Is it intentionally?

Reply:
nvmd, I misread changes in pkg/cmpstatus/base.go.